spec/lib/oidc_client_spec.rb

```diff
@@ -0,0 +1,93 @@
+RSpec.describe OidcClient do
+  subject(:client) { described_class.new }
+
+  before { stub_oidc_discovery }
+
+  describe "tokens!" do
+    before do
+      access_token = Rack::OAuth2::AccessToken.new(
+        access_token: "access-token",
+        refresh_token: "refresh-token",
+        token_type: "test",
+      )
+
+      @client_stub = stub_oidc_client(client)
+      # rubocop:disable RSpec/InstanceVariable
```

Comment:
Oh, humm, rubocop doesn't like instance allows either?
This seems like a reasonable syntax to me... I wonder why we put this rule on 🤔